beater/beatertest/monitoring.go

```diff
@@ -62,14 +63,22 @@ func AllRequestResultIDs() []request.ResultID {
 	for k := range request.MapResultIDToStatus {
 		ids = append(ids, k)
 	}
+	// add generic ids
+	ids = append(ids, []request.ResultID{
+		request.IDUnset,
+		request.IDRequestCount,
+		request.IDResponseCount,
+		request.IDResponseErrorsCount,
+		request.IDResponseValidCount}...)
 	return ids
 }
 
-// clearRegistry sets all counters to 0 and removes all registered counters from the registry
+// ClearRegistry sets all counters to 0 and removes all registered counters from the registry
```

Comment:
why this is public?

Reply:
Because it is used https://github.com/elastic/apm-server/pull/2731/files/8737271264bdbff19fb78accb631c9aee6c84f43#diff-666b27a1e49eb50fb2ec1346c9f1e52dR38